{{ cookiecutter.app_name }}/setup.py

```diff
@@ -0,0 +1,87 @@
+"""
+Install {{ cookiecutter.app_name }} via setuptools
+"""
+import sys
+
+from setuptools import setup, find_packages
+from setuptools.command.test import test as testcommand
+
+with open('test_requirements.txt') as test_reqs:
+    tests_require = test_reqs.readlines()  # pylint: disable=invalid-name
```

Comment:
that would still be a bad name, I would have to make it TEST_REQUIREMENTS or something, which I can, but since it isn't really a constant, it made more sense to me to keep it lower case.